app/src/main/res/values/strings.xml

```diff
@@ -35,6 +35,17 @@
     <string name="external_player_not_supported_yet">Playback tracking not supported in selected external player yet. Contact developers to try to add support for it.</string>
     <string name="external_player_invalid_player">No valid external player selected. Playback tracking disabled.</string>
     <string name="external_player_unknown_error">Unknown error in external player. Playback tracking disabled.</string>
+    <string name="external_player_choice">External player choice</string>
+    <string name="external_player_mpv">MPV Player</string>
+    <string name="external_player_mpv_description">A simple, small and light-weight free (as in freedom) video player. It supports a wide variety of media file formats.</string>
+    <string name="external_player_mx_player_free">MX Player Free</string>
+    <string name="external_player_mx_player_free_description">Powerful video player with advanced hardware acceleration and subtitle support. Ads everywhere.</string>
```

Comment:
The "Ads everywhere." part sounds a bit offensive to the developer

Reply:
Yeah, I wouldn't include any info about ads at all, honestly.